### PR TITLE
docs: add details about NPM scripts completion

### PR DESCRIPTION
**Pull request by edouard-lopez**
* revamp videos layout
* add details about NPM scripts completion

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,15 +34,16 @@ _You may also like:_
 - Provide completion for `pnpm remove` command, even in workspace's packages (by specifying `--filter` option).
 - Provide completion for npm scripts in `package.json`.
 
-Note that we won't provide completion for all commands and options,
-and we only focus those frequently used.
-Too many completion items may have impact on efficiency.
+### Limitiation
+
+* Note that we won't provide completion for all commands and options, and we only focus those frequently used. Too many completion items may have impact on efficiency.
+* NPM scripts completion requires the presence of a `name` property in the `package.json` file.
```
Comment (g-plane) [inline README.md:40]:
```suggestion
- Note that we won't provide completion for all commands and options, and we only focus those frequently used. Too many completion items may have impact on efficiency.
- NPM scripts completion requires the presence of a `name` property in the `package.json` file.
```

**Comment by g-plane** [comment]
Videos look too small to watch.

**Comment by edouard-lopez** [comment]
Agree, is there a real value to have videos for every shell, though? A single one focus on the terminal (without desktop, window's decoration, etc.) would be more useful. For my projects, I use [peek](https://github.com/phw/peek) to do such recording.

**Comment by edouard-lopez** [comment]
Removed the changes related to the video

**Review by g-plane — APPROVED** [review]
Thanks!

**Pull request merged**